src/rgw/rgw_keystone.cc

```diff
@@ -293,3 +298,136 @@ void RGWKeystoneTokenCache::invalidate(const string& token_id)
   tokens_lru.erase(e.lru_iter);
   tokens.erase(iter);
 }
+
+
+#if 0
```

Comment:
Hmm, I think it's because the code movements have been accomplished in separated commits without any further modifications. Those things should be corrected later in commits without the *rgw: ONLY move (...)* prefix.